pkg/diff/diff.go

```diff
@@ -501,9 +502,9 @@ func compareData(map1, map2 map[string]*dbutil.ColumnData, orderKeyCols []*model
 		}
 		equal = false
 		if data1.IsNull == data2.IsNull {
-			log.Errorf("find difference data in column %s, data1: %s, data2: %s", key, data1.Data, data2.Data)
+			log.Error("find difference data", zap.String("column", key), zap.Reflect("data1", map1), zap.Reflect("data2", map2))
```

Comment:
log `map1`/`map2` rather than `data1.Data`/`data2.Data`?

Reply:
old log only print the column's data, not convient for select the line, so I modify this log